Lib/test/test__xxsubinterpreters.py

```diff
@@ -8,12 +8,15 @@
 import threading
 import time
 import unittest
+import platform
 
 from test import support
 from test.support import script_helper
-
+from ctypes import c_void_p, sizeof
 
 interpreters = support.import_module('_xxsubinterpreters')
+AIX = platform.system() == 'AIX'
+ABI = sizeof(c_void_p) * 8
```

Comment:
When we're talking about the C-API, "ABI" has a specific meaning, so a better name for this might be `POINTER_SIZE`.

Also, I'd recommend against using ctypes here.  You can get the platform bitness (i.e. size of a pointer) elsewhere in the stdlib.  For example, you could use [`struct.calcsize()`](https://docs.python.org/3/library/struct.html#struct.calcsize), with a [format string of `'P'`](https://docs.python.org/3/library/struct.html#format-characters):

```python
import struct
struct.calcsize('P')  # should return 4 for 32-bit OS, 8 for 64-bit
```

You could use [sys.maxsize](https://docs.python.org/3/library/sys.html#sys.maxsize) too.

Finally, you probably don't need a constant for this (or `AIX`).  Since it's only used in one spot I'd recommend that you move the line there:

```python
    is32bit = sys.maxsize < 2**31
    if platform.system() == 'AIX' and is32bit:
        ...
```